### DISCUSSION TITLE
Zipkin-instrumentation

### DISCUSSION THREAD

**Discussion by mitoeth** [open-telemetry/opentelemetry-cpp#1861] [Q&A]
Hello, trying to understand the OTEL-setup with Zipkin exporter without knowing much about C++

I can see the instructions on how to install the Zipkin exporter: https://github.com/open-telemetry/opentelemetry-cpp/tree/main/exporters/zipkin

Does this also make sure that the application is instrumented? Or do you still need to do that manually like described here: https://github.com/flier/zipkin-cpp/blob/master/docs/GettingStarted.md

Thanks

**Comment by lalitb** [comment]
Please go through the example here to instrument the application and export traces to Zipkin:

https://github.com/open-telemetry/opentelemetry-cpp/tree/main/examples/zipkin
